### DISCUSSION TITLE
game left 4 dead 2 run with lgsm cfg "command not found"

### DISCUSSION THREAD

**Discussion by Kaze1027** [GameServerManagers/LinuxGSM#4068] [Q&A] [answered]
I run l4d2 game server with LinuxGSM, I added commands into l4d2server.cfg, when the log print:
```
/home/l4d2server/lgsm/config-lgsm/l4d2server/l4d2server.cfg: line 1: motdfile: command not found
```
Well when I use 'motdfile' in console, it works. Just not work when it is in the cfg file.
![image](https://user-images.githubusercontent.com/51438620/209829787-46d51d91-01a0-46ab-8673-555f22944511.png)
![image](https://user-images.githubusercontent.com/51438620/209829901-e05deb31-ebe6-46c9-a065-8cd369ad188d.png)
![image](https://user-images.githubusercontent.com/51438620/209830065-3b7f9303-1fe6-4bb7-802b-c6a6db3967e1.png)
![image](https://user-images.githubusercontent.com/51438620/209830194-aceefa87-799e-4a21-a949-9fa3bfc015b7.png)

**Comment by Bobbin-hash** [comment]
Is l4d2server working ok for you? I've installed it, it starts and run, but having issues connecting to server, as the application wont accept the start parameter +servercfgfile ${servercfg} . So removed it and then needed to add a server.cfg to the server files directory for the application to start. LGSM then created a copy and placed l4d2server.cfg in the same directory. Everything starts, ports are open & this is the 10th server I've worked on (all others worked ok) and it just wont play ball. It starts & I can change maps etc and enter commands from the console, but in the log states prevented spawning and server is listening but cannot connect to it even thou ports are open (as per all other working servers like tf2, csgo etc...)

**Comment by Kaze1027** [comment, marked as answer]
Thx for your answer, I've got how to configure it. 
First of all, this cfg file (/home/l4d2server/lgsm/config-lgsm/l4d2server/l4d2server.cfg) is worked for LGSM, not the game server program.
All the convar you should write it into this cfg file ( serverfile/left4dead2/cfg/l4d2server.cfg) .
